src/modules/listPost.js

```diff
@@ -0,0 +1,16 @@
+// display
+const listPost = (list) => {
+  const dataContainerUl = document.querySelector('.scores__line');
+  if (!list) {
+    dataContainerUl.innerHTML = 'No post available 😞';
+    return;
+  }
+
```

Comment:
- [ ] Kindly add this line ` dataContainerUl.innerHTML = '';` ,so when I refresh I got the scores withoud repeating